app/controllers/SwitchOverErrorController.scala

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package controllers
+
+import config.FrontendAppConfig
+import controllers.actions.{AuthIdentifierAction, UserDataRetrievalAction}
+import play.api.mvc.{Action, AnyContent, MessagesControllerComponents}
+import views.html.SwitchOverErrorView
+
+import javax.inject.Inject
+import scala.concurrent.Future
+
+
+class SwitchOverErrorController @Inject()(
+    identify: AuthIdentifierAction,
+    getData: UserDataRetrievalAction,
```

Comment:
now I think about it, do we even need a the UserDataRetrievalAction in there? happy to keep it, but there's a few places where we don't need it I think

Reply:
updated now